api/diagnostics_test.go

```diff
@@ -386,7 +386,7 @@ func (s *DiagnosticsTestSuit) TestRunSnapshot() {
 	s.assert.Equal(responseJSON.Status, "Job has been successfully started")
 	s.assert.NotEmpty(responseJSON.Extra.LastBundleFile)
 	time.Sleep(2 * time.Second)
-	snapshotFiles, err := ioutil.ReadDir("/tmp/snapshot-test")
+	snapshotFiles, err := ioutil.ReadDir(DiagnosticsBundleDir)
 	s.assert.NoError(err)
 	s.assert.True(len(snapshotFiles) > 0)
 
```

Comment:
any change to `*_test.go` should be a separate commit.

Reply:
Let me see what i can do here....